### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by wbentley15**
First draft of the OSIC update blog.

Bios:
Walter Bentley – Rackspace Private Cloud Solutions Architect – Walter is a new Racker with a diverse background in Production Systems Administration and Solutions Architecture. He brings over 15 years of experience across numerous industries such as Online Marketing, Financial, Insurance, Aviation, Food Industry and Education. In the past, always being the requester, consumer and advisor to companies to use technologies such as OpenStack, now promoter of OpenStack technology and Cloud educator.

Twitter: @djstayflypro
LinkedIn: http://goo.gl/r2p21i
Github: wbentley15

Accept to allow redaction, editing or structure changes by technical writer staff desired date of publishing
Publish date: 01-22-2016

### CODE REVIEW

**Comment by wbentley15** [comment]
@catlook Hello...long time no submit blog for your wonderful review.  You mind doing your magic as always.  Thank you in advance for your help.  BTW my book is finally being published at the end of this month, super excited!!!

**Comment by wbentley15** [comment]
@rgbkrk Dude I have no clue why the automated checks failed.  Let me know what changes I need to make.  Thanks!

**Comment by rgbkrk** [comment]
That's @ktbartholomew's handy "we have enough new topics" checker. It's alerting on the fact that there is only one post with some of the tags you have here:

```
Category 'osic' only has 1 post
Category 'openstack-innovation-center' only has 1 post
Category 'rackspace' only has 1 post
Category 'intel' only has 1 post
```

Couple suggestions:
* Make the rackspace category be Rackspace
* Choose either OSIC or openstack-innovation-center

The build will still "fail" on that, though those certainly need unification there. I'll let the Keith decide what to do next as our resident taxonomist.

**Comment by wbentley15** [comment]
Fixed the categories per your suggestions.  It took me a few minutes to figure out why the build was failing :D.  I went with OSIC and it should be added to the category list as I have more updates coming down the trail.  Thanks!

**Comment by rgbkrk** [comment]
Looks good. Thanks for helping us combat the topic sprawl.

**Comment by catlook** [comment]
Congrats on the book…that’s really impressive :)

-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting – the #1 Managed Cloud Company
DevDocs on Developer Experience Team
Office: (210) 312-8193   Cell: (210) 364-1060
cat.lookabaugh@rackspace.com

We should not write so that it is possible for the reader to understand us, but so that it is impossible for him to misunderstand us. - Quintilian, rhetorician (c. 35-100)

Harmony + Learner + Responsibility + Context + Intellection + Arranger + Consistency + Relator + Input + Analytical

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+published: true
+categories:
+    - OSIC
+    - Rackspace
+    - Intel
+    - rackspace-private-cloud
+    - OpenStack
+---
+
+Since the initial launch of the OpenStack Innovation Center back in July of 2015 much work has been done.  Wanted to take a moment to share the current status and some details about its next phases.  If you are unfamiliar with OSIC let me start off with some very quick background information.
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: ... in July of 2015, much work ...
... unfamiliar with OSIC, let me start ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+<!-- more -->
+
+The OpenStack Innovation Center, abbreviated to ‘OSIC’, is a join partnership between Rackspace and Intel to further accelerate the development of enterprise capabilities into upstream OpenStack code.  It is intended to bring together Rackspace and Intel engineers in order to:
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: ... is a joint partnership ...
... Intel implented to further ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+>***“advance the scalability, manageability and reliability of OpenStack by adding new features, functionality and eliminating bugs through upstream code contributions”*** 
+
+as mentioned in the [Rackspace press release](https://blog.rackspace.com/newsarticles/rackspace-collaborates-with-intel-to-accelerate-openstack-enterprise-feature-development-and-adoption/).
+
+It is also intended to offer the greater OpenStack community with an OpenStack cloud at an enterprise scale in order to develop/test that new OpenStack functionality.  The OSIC hub can be found within Rackspace’s corporate headquarters in San Antonio, TX nicknamed “Castle”. Personally the OSIC launch was a very exciting one since it was aimed at solving some of OpenStack’s greatest challenges thus far, which are enterprise focused features and the basic idea that OpenStack was not enterprise ready.  Those two 1,000 node OpenStack clusters sort of dispel that second wrong idea in so many ways.  OpenStack runs in my blood and I am very proud to be part of the community.
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: It is also intended to offer an OpenStack cloud (at an enterprise scale) to the greater OpenStack community in order to develop/test that new OpenStack functionality. [Ed. Note: wasn't entirely sure what to do with this sentence.  If my revision doesn't capture your intent, ping me.  We'll craft it better!]

... Personally, I thought the OSIC launch was a very exciting one. I was ...
... my blood, and I am  ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+OK, enough of my banter and lets get down to business with some updates!
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: ... banter - let's ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+**Innovation Center**
+
+Yes, you can actually see, touch and sit in it now.  The Innovation Center located at Rackspace’s Castle launched on September 10th of 2015.  Imagine reserving a whole wing of what used to be a old shopping mall just to seat a bunch of OpenStack developers, engineers, support and architect staff.  This is the location where Rackspace and Intel engineers can collaborate on all things OpenStack.  We openly invite any and all who are interested to come on down to see it with there own eyes.
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: ... touch, and sit ...
... Center, located at Rackspace’s Castle, launched ...
... support, and architect staff ... [Ed question: architect?  or architecture? ]
... This location allows Rackspace and Intel engineers to collaborate ...
...  with their own ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+**Developer Training/Joint OpenStack Engineering**
+
+As of October 2015 Intel has placed ~12 developers into the Innovation Center to take part in the detailed training program created in order to on-board OpenStack focused developers.  From that point forward roughly 10-15 additional developers will be brought in to work in the Innovation Center on a monthly basis.  The details around the OSIC engineering road map can be found in the below slide from the Tokyo OpenStack Summit:
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: As of October 2015, Intel has ...
...  point forward, roughly ...
... found in the following slide ...  [Ed note: Picky, I know, but, per our Style Guide,  we prefer 'following' and 'preceding' as opposed to 'below' and 'above'.  FYI :) ]

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+![OSIC Roadmap](http://www.hitchnyc.com/content/images/2016/01/osic-roadmap-n.png)
+
+**Developer Cloud**
+
+Come on...you know you only read this far to get the scope on this section.  Being honest, the Developer Cloud is pretty darn cool and I can not blame you.  Just saying “two 1000-node OpenStack clusters” out loud sounds impressive.  As an architect my first thought was “show me the details!”.  Well you will be the first to get a look at the details that make up the first of the 1000-node cluster that is hosted at the Rackspace Dallas Fort Worth datacenter.  Could not think of a better combination of being funded by Intel and supported by Rackspace.
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: ... the scoop in this section ...
... Being honest, you're thinking the Developer Cloud is pretty darned cool, and I can not blame you. ...
.. As an architect, my first ...
... Well, you are the first ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+ 
+Keep in mind the below details encompass only the first 1000-node cluster that is ready for use and hosted at Rackspace.  Unfortunately, I was not able to get any exciting details on the second cluster located at Intel in Oakland, California yet.  Keep posted for more updates to come at a later date.  Before we jump in I want to thank my fellow Racker, ***James Thorne ([@jameswthorne](https://twitter.com/jameswthorne))***, for providing the cloud details.  He is our resident keeper and protector of the Rackspace hosted OpenStack Developer cloud.
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: Keep in mind that the following details encompass only ....
... Before we jump in, I want ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+
+***OSIC Cluster @ Rackspace***
+
+* Made up of ~1012 bare metal servers that are broken up in particular ways to server the various requests that come in from the OpenStack community.
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: ... ways to serve the various ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+* Over 60% of the cluster is however running OSA
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: ... cluster is running ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+* The remainder of the cluster is configured for server bare metal server environments as per the community requests thus far.
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: ... environments as requested by the community. ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+* Each cabinet has two top-of-rack Cisco Nexus 3172-PQ network switches.
+* Each cabinet also has one top-of-rack network switch for out-of-band management.
+
+The hardware is broken out into two cloud regions and the details are below:
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: ... and the details follow: ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+    22x Compute nodes with spindle drives
+    4x Controller nodes
+    5x Network nodes
+    1x Util node (Running HAProxy right now)
+    1x Jump node
+
+
+**Who’s using the Developer Cloud?**
+
+Currently the Intel Bare Metal CI Team and Mirantis are the only OSIC customers using the cluster as of today.  Both of those teams are using the bare metal server option as mentioned above.  The next consumers slated to come on board will the the OpenStack Foundation Infrastructure team and moving the OSIC sign-up form from this [link](http://go.rackspace.com/developercloud) to http://www.osic.org, which will sit on the Developer cloud as well.
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: ... Both of those teams use the bare metal ...
... consumer slated to come on board is the OpenStack Foundation Infrastructure team. We're also moving the OSIC sign-up form from this [link](http://go.rackspace.com/developercloud) to http://www.osic.org, which will sit on the Developer cloud.

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+
+We will close out this update with two pretty pictures.  Rather confident that you like myself wishes that this was a screenshot of Horizon from their home labs.  We can all dream right?  Please stay tuned for more updates to come and keep Stacking!
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: We close out this ...
... I'm rather confident that you wish that this was a screenshot of Horizon from their home labs, just like I do. ...

File: _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md [hunk 15]
```diff
@@ -0,0 +1,120 @@
...
+
+{<2>}![Rackspace OSIC Cloud Region 1](http://www.hitchnyc.com/content/images/2016/01/osic-cloud1-n.png)
+{<3>}![Rackspace OSIC Cloud Region 2](http://www.hitchnyc.com/content/images/2016/01/osic-cloud2-n.png)
+
+Check out the last 8 minutes of the recorded OSIC session from the Tokyo OpenStack summit back in October 2015 to see the video of the OSIC launch at Castle: [https://www.openstack.org/summit/tokyo-2015/videos/presentation/intel-cloud-for-all-openstack-innovation-center](https://www.openstack.org/summit/tokyo-2015/videos/presentation/intel-cloud-for-all-openstack-innovation-center).
```
Comment (catlook) [inline _posts/2016-01-22-openstack-innovation-center-updates-january-2016.md]: [Ed note: Very cool article...We should all be proud of OSIC!  -- Cat]

**Comment by wbentley15** [comment]
@catlook thank you for all these great suggestions.  I have updated the article accordingly.

**Comment by wbentley15** [comment]
@rgbkrk Ship it whenever you are ready fine sir!

**Comment by rgbkrk** [comment]
Alright, the powers that be have spoken.

> ​*Get critical mass before creating new categories*​. If you want an OSIC category, you needs to write (or find) about 7 posts that would ostensibly have that that category, ​_then_​ add the category to those posts (on the last PR).

**Comment by wbentley15** [comment]
Ok I will drop the OSIC category under direst :D.

Walter

**Comment by rgbkrk** [comment]
You are, as always, awesome to work with. :bow:

**Comment by wbentley15** [comment]
@rgbkrk It should be good to go now :+1:

**Comment by wbentley15** [comment]
@rgbkrk I do believe it is ready to officially ship now!

**Comment by wbentley15** [comment]
@rgbkrk any way we can ship today?  I promised RPC marketing it would be out this week, sorry for the pressure on a Friday.

**Comment by rgbkrk** [comment]
:shipit:

**Pull request merged**